playground/doc/playground.mdx

```diff
@@ -1,40 +1,90 @@
 import React from 'react';
 import { mdx, MDXCreateElement } from '@mdx-js/react';
 import { Props, Description } from '~/props/dist/index';
-import { Playground } from '../dist';
+import { Playground, ShadowPlayground } from '../dist';
 import { Button } from '../test/index';
+import styles from '../test/styles.css';
 import { CssLayout } from '~/mdx-layout-css';
 export default CssLayout;
 
-# Playground Component
+## Playground Component
 
 <Description of={Playground} />
 
-## Usage
+### Usage with global styles
+
+This includes the usage with most CSS-in-JS libraries which rely on global style tag.
 
 ```tsx
-import { Playground } from '@divriots/dockit-react';
+import { Playground } from '@divriots/dockit-react/playground';
 import { Button } from './Button';
 
 <Playground
   scope={{ Button }}
   code={`
-  <Button>Button</Button>
-  <Button primary>Primary</Button>
-  <Button danger>Danger</Button>
+<Button>Button</Button>
+<Button primary>Primary</Button>
+<Button danger>Danger</Button>
   `}
 />;
 ```
 
 <Playground
   scope={{ Button }}
   code={`
-<Button>Button</Button> 
-<Button primary>Primary</Button> 
-<Button danger>Danger</Button> 
-`}
+<Button>Button</Button>
+<Button primary>Primary</Button>
+<Button danger>Danger</Button>
+  `}
 />
 
-## Props
+### Playground Props
 
 <Props of={Playground} />
+
+## ShadowPlayground Component
+
+<Description of={ShadowPlayground} />
+
+### Usage with Shadow DOM
+
+```tsx
+import { ShadowPlayground } from '@divriots/dockit-react/playground';
+import styles from './styles.css';
+
+<ShadowPlayground
+  styles={styles}
+  code={`
+<button className="btn">Button</button>
+<button className="btn btn-primary">Primary</button>
+<button className="btn btn-danger">Danger</button>
+  `}
+/>;
+```
+
+<ShadowPlayground
+  styles={styles}
+  code={`
+<button className="btn">Button</button>
+<button className="btn btn-primary">Primary</button>
+<button className="btn btn-danger">Danger</button>
+  `}
+/>
+
+### Recommended setup for your Design System
+
+You can setup a wrapper component to use for Design System demos in the following manner:
+
+```tsx
+import { ShadowPlayground } from '@divriots/dockit-react/playground';
+import styles from './styles.css';
+export const DSPlayground = (props) => {
+  return <ShadowPlayground styles={styles} {...props} />;
+};
+```
+
+Then just import and use `DSPlayground` to avoid setting up styles in every demo.
```

Comment:
@muryoh this section `Recommended setup for your Design System` is my last addition for AOS, can you please have a look?

![image](https://user-images.githubusercontent.com/137844/128047714-414e121d-85b0-45f4-94f6-c0ec182354c6.png)

https://backlight.dev/edit/wzPn6NO54tDJlBxCtjGx/playground/stories/index.stories.tsx?branch=~feat/playground-shadow-dom@44v7XI9EE5hzBwU7oxi8dbGjKz03

Reply:
Looks good to me

I think very often people will actually use the starterkits as doc though, so maybe we could update them?

Reply:
noted: https://github.com/divriots/webcomponents-editor/issues/2525